### DISCUSSION TITLE
How to flex "Update all dependencies" pr/commit title to add some uniqueness

### DISCUSSION THREAD

**Discussion by mattnelson** [renovatebot/renovate#21562] [Suggest an Idea]
### Type of discussion.

I'm proposing an idea

### Tell us more.

I am using renovate to update all dependencies weekly. This results in a new PR/commit with the title `Update all dependencies` each week. After a few weeks the commit history has a bunch of commits with the same title and searching previous PRs all show the same title. This is creating a growing scenario where the commit history starts to lose meaning and it is difficult to distinguish them apart. Are there options to configure a unique suffix on these titles to bring some uniqueness? I took a look through the docs and did not connect any dots that something like this was currently possible.

What I'm envisioning is an option to dynamically generate a contextual suffix. In the below example it could be time based from an iso8601 format

```json
  "schedule": [
    "every weekend"
  ],
  "commitMessageSuffix": "for {now:YYYY.ww}"
```

Example 
```
Update all dependencies for 2023.15
Update all dependencies for 2023.16
```

**Comment by rarkins** [comment]
What would be the desired behavior if you leave the PR open for so long that it's a new week?

**Reply by mattnelson** [reply]
I'm not sure. We haven't had that happen yet. I'm not even sure what renovate does in that scenario? Does it close and open a new PR? Update the existing one? nothing?

**Reply by rarkins** [reply]
If a PR title changes then Renovate would rename the existing PR in place.
